README.md

```diff
@@ -90,16 +90,15 @@ Add the dependencies into the project's `build.gradle`
 def arrow_version = "0.9.1-SNAPSHOT"
 dependencies {
     compile "io.arrow-kt:arrow-core-data:$arrow_version"
```

Comment:
Given that core now includes core-data too, I'd say remove the half-versions from here.